modules/linear_boltzmann_solvers/a_lbs_solver/lbs_solver.cc

```diff
@@ -248,10 +255,10 @@ LBSSolver::GetUnitCellMatrices() const
   return unit_cell_matrices_;
 }
 
-const MeshContinuum&
-LBSSolver::Grid() const
+const std::map<uint64_t, UnitCellMatrices>&
```

Comment:
Yes it is used in quite a few places. Mainly for DoF ids, global ids, etc. The return type is the same as the member type this is an accessor for. 

I think it would be good to discuss standardizing the types we use throughout the code base at some point though. We should be consistent on when we use what types.